pkg/sidecar/inject.go

```diff
@@ -0,0 +1,238 @@
+/*
+Copyright 2019 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sidecar
+
+import (
+	"fmt"
+	"log" // TODO(spencer-p) use zap
+	"os"
+	"strconv"
+	"strings"
+
+	"knative.dev/pkg/apis"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	IMAGE_KEY    = "K_SOURCE_ADAPTER_IMAGE"
+	ADAPTER_PORT = 38080
```

Comment:
this should be `DEFAULT_ADAPTER_PORT` right?

Reply:
Correct